Example/UINotifications-Example/ViewController.swift

```diff
@@ -47,6 +47,10 @@ enum NotificationStyle: UINotificationStyle {
             return #colorLiteral(red: 1, green: 0.431372549, blue: 0.431372549, alpha: 1)
         }
     }
+
+    var thumbnailSize: CGSize {
+        return .init(width: 100, height: 100)
+    }
     
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Example/UINotifications-Example/ViewController.swift;
  Line: 54;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)